.travis.yml

```diff
@@ -1,7 +1,6 @@
 dist: xenial
 language: python
 env:
-  - ONNX_VERSION="==1.1.2" ONNX_ML=1
```

Comment:
This backward combability is required. Is there any thing blocked this version?

Reply:
No, I just wanted to have faster CI. I'll add it back.

Reply:
haven't seen it was added back yet, or do I miss something?

Reply:
It just wanted to keep the test short:
python 3.5 - tests against onnx==1.2.3
python 3.6 - tests against onnx==1.3.0
python3.7 - test against onnx (latest version)

It is specified below in the file.

Reply:
I mean ONNX_VERSION == 1.1.2 need be tested as well since there is backward compatible requirement for this version.